tensorflow/lite/delegates/gpu/BUILD

```diff
@@ -159,6 +162,15 @@ ios_static_framework(
     deps = [":metal_delegate"],
 )
 
+macos_dylib(
```

Comment:
Can you add a comment here about how to build, if any special flags are required? And maybe include something like "Note: Support for MacOS is best-effort at the moment.".

Reply:
@jdduke ok, adding recommended build options in the comment